tests/throughput_test/bandwidth_test.c

```diff
@@ -10,14 +10,21 @@
  * If you wish to obtain a commercial license, please contact Mellanox at support@mellanox.com.
  */
 
-
+/*
+ * How to build:
+ * gcc -g -Wall -DVMA_ZCOPY_ENABLED=1 -o bandwidth_test bandwidth_test.c -lrt
+ *
+ * Server mode ./bandwidth_test -s -i 224.4.4.1
+ * Client mode ./bandwidth_test -c -i 224.4.4.1
+ */
```

Comment:
Could you add more details:
- how to compile using Berkeley API
- how to compile using vma_poll
Pay attention that probably -I and -L are needed to avoid issue when libvma.so installed at unusual location